README.md

```diff
@@ -130,8 +130,8 @@ strategies:
 
 ### RemoveDuplicates
 
-This strategy makes sure that there is only one pod associated with a Replica Set (RS),
-Replication Controller (RC), Deployment, or Job running on the same node. If there are more,
```

Comment:
@damemi are you fine with removing `Deployment` from the doc?

Reply:
I would prefer to clarify that it uses the Deployment->RS relationship (that works right?). Could be a note after this paragraph or something, but not including it at all doesn't seem completely accurate to me

Reply:
Add Deployemnts back into the docs in commit 7e389cc7cc676ade4dbdedf4e456925bbf5db5cf